specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json

```diff
@@ -369,7 +389,49 @@
           "default": {
             "description": "An error response received from the Azure Data Factory service.",
             "schema": {
-              "$ref": "#/definitions/ErrorResponse"
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataFactory/factories/{factoryName}/upgrade": {
```

Comment:
Upgrade should be modeled as a PUT. @hvermis - I dont recall we discussing this in our conversation.

Reply:
Removed from swagger for now as per offline discussion.